proto/api.proto

```diff
@@ -24,10 +24,6 @@ import "cockroach/proto/data.proto";
 import "cockroach/proto/errors.proto";
 import "gogoproto/gogo.proto";
 
-option (gogoproto.sizer_all) = true;
-option (gogoproto.marshaler_all) = true;
-option (gogoproto.unmarshaler_all) = true;
```

Comment:
Someone doing optimization work might be grep'ing for whether we specify the `sizer` or `sizer_all` options. Or even just wanting to determine what gogoproto options we're using.